### ISSUE TITLE
Blob doesn't kill people when they are blobbed

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#36367]
(WEB REPORT BY: adacovsk REMOTE: 172.93.101.77:7777)
# Revision
d76e75a35896c2d0bdcb27aa940decc70d5014ba

# Description
Blob someone, they scream on the radio and break free and start killing again
# Steps to Reproduce

# What you Expected

# What Actually Happened

**Comment by kilozombie** [comment]
very hard to parse this one

**Comment by DeityLink** [comment]
What did he mean by this?

**Comment by SECBATON-GRIFFON** [comment]
people die when they are blobbed

**Comment by west3436** [comment]
Needs more info.
